aptos-move/block-executor/src/executor.rs

```diff
@@ -714,6 +787,32 @@ where
         aggregator_v1_delta_writes
     }
 
+    /// Can be called (at most) once after transaction is committed to internally include
+    /// the resource group updates & delta outputs with the transaction outputs.
+    fn combine_groups(
```

Comment:
nit: to make naming consistent, should we call this materialize_group or serialize_group?

Reply:
changed to serialize / serialized